File: Taller Tipos de clases/C#/Clase Interfaz/Servicio.cs
interface IServicio
{
    void Activar();
}

class Wifi : IServicio
{
    public string Nombre;

    public Wifi()
    {
        Nombre = "Sin nombre";
    }

    public Wifi(string nombre)
    {
        Nombre = nombre;
    }

    public Wifi(Wifi w)
    {
        Nombre = w.Nombre;
    }

    public void Activar() { }
}

File: Taller Tipos de clases/JAVA/Clase Interfaz/Servicio.java
interface Servicio {
    void activar();
}

class Wifi implements Servicio {
    String nombre;

    public Wifi() {
        nombre = "Sin nombre";
    }

    public Wifi(String nombre) {
        this.nombre = nombre;
    }

    public Wifi(Wifi w) {
        this.nombre = w.nombre;
    }

    @Override
    public void activar() {}
}

